src/component/Button.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+
+const Button = ({ color='#8888b9', label, onClick, isClicked=false, isDisabled=false}) => {
```

Comment:
It is better to keep optional props at last.
`const Button = ({ label, onClick, color='#8888b9', isClicked=false, isDisabled=false}) => {`